doc/nrf/libraries/others/network_core_monitor.rst

```diff
@@ -0,0 +1,122 @@
+.. _network_core_monitor:
+
+Network core monitor
+####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This library monitors the network core status of the nRF5340 processor.
+
+Overview
+********
+
+This library monitors the network core status of the nRF5340 processor by application core.
```

Comment:
Maybe you could leave this line out as it is pretty close to the description before the overview.